web/src/components/button.js

```diff
@@ -41,4 +62,6 @@ Button.propTypes = {
     isPrimaryDark: PropTypes.bool,
     isSecondary: PropTypes.bool,
     isSecondaryDark: PropTypes.bool,
+    to: PropTypes.string.isRequired,
+    href: PropTypes.string.isRequired,
```

Comment:
Are these both `required` or will it be an either/or?

Reply:
@asasmith either/or depending on how the component is being used

Reply:
@asasmith I was steering away from passing default props since its a conditional component, but if youre thinking we should, what are your thoughts on the default prop here? An empty string?

Reply:
I'm concerned that we'll see errors if there are required props that aren't passed into this component.

To meet the conditions of the component an empty string will evaluate to false.